dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -112,6 +117,49 @@ def docker_compose_tests(
     sys.exit(return_code)
 
 
+def run_tests(
+    shell_params: ShellParams, dry_run: bool, verbose: bool, extra_pytest_args: Tuple
+) -> Tuple[int, str]:
+    env_variables = get_env_variables_for_docker_commands(shell_params)
+    perform_environment_checks(verbose=verbose)
+    cmd = ['docker-compose', 'run', '--service-ports', '--rm', 'airflow']
+    cmd.extend(list(extra_pytest_args))
+    test_result = run_command(
+        cmd,
+        verbose=verbose,
+        dry_run=dry_run,
+        env=env_variables,
+    )
+    return (
+        test_result.returncode,
+        f"Running tests for Python {shell_params.python}",
+    )
+
+
+def run_tests_in_parallel(
+    shell_params_list: List[ShellParams],
+    python_version_list: List[str],
+    parallelism: int,
+    dry_run: bool,
+    verbose: bool,
+):
+    """Run tests in parallel"""
+    get_console().print(
+        f"\n[info]Running tests with parallelism = {parallelism} "
+        f"for the python versions: {python_version_list}[/]"
+    )
+    pool = mp.Pool(parallelism)
```

Comment:
Why not use [Joblib](https://joblib.readthedocs.io/en/latest/)? Gives a lot more options...

Reply:
We don't need those more options, really. We used to use "gnu parallell" which was also a parallell combo but adding it caused enormous complexity - because what we essentially need is to run n jobs in parallel and wait for them.

Reply:
Airflow has 600+dependencies adding one more seems like not much but we should avoid it if possible.

Reply:
BTW @chethanuk-plutoflume - what "more options" did you have in mind that woudl be useful? Curious to hear - there are always interesting things to learn from other's experiences.